File: Codewars_C/8kyu/ex06/basic_op.c
#include <stdlib.h>
#include <stdio.h>

int basic_op(char op, int value1, int value2)
{
    switch (op)
    {

    case '+':
        return value1 + value2;
    case '-':
        return value1 - value2;
    case '*':
        return value1 * value2;
    case '/':
        if (value2 == 0)
        {
            fprintf(stderr, "Error: Division by zero is not allowed\n");
            exit(EXIT_FAILURE);
        }
        return value1 / value2;
    default:
        fprintf(stderr, "Error: Invalid operation\n");
        exit(EXIT_FAILURE);
    }
    return 0;
}

File: Codewars_Python/8kyu/ex06/basic_op.py
def basic_op(op, value1, value2):
    if op == '+':
        return value1 + value2
    elif op == '-':
        return value1 - value2
    elif op == '*':
        return value1 * value2
    elif op == '/':
        if value2 == 0:
            raise ValueError("Error: Division by zero is not allowed")
        return value1 / value2
    else:
        raise ValueError("Error: Invalid operation")